sql/database.sql

```diff
@@ -11421,7 +11421,14 @@ CREATE TABLE `form_observation` (
   `code_type` varchar(255),
   `table_code` varchar(255),
   `ob_code` VARCHAR(31) DEFAULT NULL,
-  `ob_type` VARCHAR(31) DEFAULT NULL
+  `ob_type` VARCHAR(31) DEFAULT NULL,
+  `ob_status` varchar(32) DEFAULT NULL,
+  `result_status` varchar(32) DEFAULT NULL,
+  `ob_reason_status` varchar(32) DEFAULT NULL,
+  `ob_reason_code` varchar(255) DEFAULT NULL,
+  `ob_reason_text` text DEFAULT NULL,
```

Comment:
I received a message from the text null police:
"remove the DEFAULT NULL here" :)

Also, for the other entries, I don't know the difference between 'DEFAULT NULL' or 'NULL' (and there may not be a differebce), but would choose one (NULL is used in upgrade and DEFAULT NULL is used here in database.sql)

Reply:
yep I keep forgetting:) doesn't make a diff but for consistency then.. thanks